open-metadata-implementation/access-services/information-view/information-view-api/src/main/java/org/odpi/openmetadata/accessservices/informationview/ffdc/exceptions/runtime/AddEntityException.java

```diff
@@ -0,0 +1,14 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.informationview.ffdc.exceptions.runtime;
+
+public class AddEntityException extends InformationViewUncheckedExceptionBase {
```

Comment:
I suggest renaming InformationViewUncheckedExceptionBase to InformationViewExceptionBase
It is obviously unchecked